src/test/java/io/fabric8/maven/docker/util/AuthConfigFactoryTest.java

```diff
@@ -101,7 +101,7 @@ public String decrypt(String password) {
 
     @BeforeEach
     void containerSetup() throws ComponentLookupException, SecDispatcherException {
-        Mockito.lenient().when(container.lookup(SecDispatcher.ROLE, "maven")).thenReturn(secDispatcher);
+//        Mockito.lenient().when(container.lookup(SecDispatcher.ROLE, "maven")).thenReturn(secDispatcher);
```

Comment:
Why is this mockito expectation disabled?

Reply:
Ugh i guess I forgot about those. As i remember, it happened because the Security dispatcher stuff doesn't work with never versions of maven. I guess it should be enabled, so that whoever knows how it is supposed to work can make the appropriate fixes.